### ISSUE TITLE
RCS1203 is reported for abstract attributes

### ISSUE THREAD

**Issue by hankovich** [dotnet/roslynator#1117]
**Product and Version Used**: 4.3.0

**Actual Behavior**:
```csharp
public abstract class MyAttribute : Attribute // <- diagnotic says I should add an AttributeUsageAttribute
{
}
```
